### ISSUE TITLE
Mixable food

### ISSUE THREAD

**Issue by theFox6** [Gerold55/WasteLands_Survival#62]
**Is your feature request related to a problem? Please describe.**
There are often either tons of food items or just very few.

**Describe the solution you'd like**
I'd like to have some general food item that can have different saturation (metadata) values.
Something like stew and you can throw anything inside. The more you mix the more saturating it gets.

**Describe alternatives you've considered**
Tons of food items, but hidden ones.

**Comment by olliy1or** [comment]
something like the cauldron in xdecor mod?

**Comment by Gerold55** [comment]
> something like the cauldron in xdecor mod?

Yes but you're able to have mixable food in which if you add certain food items it'll add different nutrition levels etc etc and change the look of the food also.
